README.md

```diff
@@ -15,6 +15,16 @@
 
 Now you can visit `localhost:4000` from your browser.
 
+## Automatic code reloading
+
+1. Install the `nodejs package` (or your system's equivalent) if you don't have it already
+1. Install npm dependencies with `npm install`
+1. Install `inotify-tools` with the package manager of your preference:
```

Comment:
I don't remember to install inotify-tools is it something specific of Arch linux?